blaze/compute/tests/test_postgresql_compute.py

```diff
@@ -212,3 +228,74 @@ def test_join_type_promotion(sqla, sqlb):
     result = set(map(tuple, compute(expr, {t: sqla, s: sqlb}).execute().fetchall()))
     expected = set([(1, 'a', 'a'), (1, None, 'a')])
     assert result == expected
+
+
+def test_dist(nyc):
```

Comment:
This data also presents a real use case for some of the less-often-seen math functions like `atan2`